internal/build/query_builder.go

```diff
@@ -88,20 +88,22 @@ type Select struct {
 }
 
 type Repo struct {
-	Source      string   `yaml:"source"`
-	Destination string   `yaml:"destination"`
-	PackageName string   `yaml:"package_name"`
-	StructName  string   `yaml:"struct_name"`
-	TableName   string   `yaml:"table_name"`
-	DBLibrary   string   `yaml:"db_library"`
-	Test        bool     `yaml:"test"`
-	Select      []Select `yaml:"select"`
+	Source       string   `yaml:"source"`
+	Destination  string   `yaml:"destination"`
+	DatabaseName string   `yaml:"database_name"`
```

Comment:
Please Rename it to `Dialect`. It needs to change other files too.
Also, It's better to define a new type for dialect and define some constance variables of your type that refer to all dialects of `goqu`.